CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 - [#985](https://github.com/airblade/paper_trail/pull/985) - Fix RecordInvalid
   error on nil item association when belongs_to_required_by_default is enabled.
+- [#988](https://github.com/airblade/paper_trail/pull/988) - Fix ActiveRecord
+  version check in `VersionConcern` for Rails 4.0
```

Comment:
Please put your changelog entry under Unreleased -> Fixed. Here you have it under 7.1.2 -> Fixed. Thanks.